aspnetcore/grpc/netstandard.md

```diff
@@ -66,8 +66,8 @@ For more information, see [Configure gRPC-Web with the .NET gRPC client](xref:gr
 
 Requirements and restrictions to using `WinHttpHandler`:
 
-* Windows 10 Build 19622 or later.
-* A reference to [System.Net.Http.WinHttpHandler](https://www.nuget.org/packages/System.Net.Http.WinHttpHandler/) version 6.0.0-preview.3.21201.4 or later.
+* Windows 10 Build 19622 or later. May require the use of a [Windows Insider](https://insider.windows.com) build).
```

Comment:
```suggestion
* Windows 10 Build 19622 or later. May require the use of a [Windows Insider](https://insider.windows.com) build.
```

Reply:
I didn't do it because the destination is the program and not really a build, but that's fine ... I'll do it on my local and shoot it in.

Reply:
Oh ... sorry ... I thought u were placing the word "build" differently. I see ..... **a stray paren!** 😈